File: src/main/java/com/hugo/Guess/Game.java
package com.hugo.Guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        Scanner scanner = new Scanner(System.in);

        int secrect = random.nextInt(10) + 1;
        int answer;
        System.out.print("Plz input your number : ");
        answer = scanner.nextInt();
        while (answer != secrect) {
            if (answer > secrect) {
                System.out.println("Lower");
            } else {
                System.out.println("Hight");
            }
            System.out.print("Plz input your number : ");
            answer = scanner.nextInt();
        }
        System.out.println("Congratulation Answer is :" + answer);

    }

}

File: src/main/kotlin/com/hugo/Guess/Game.kt
package com.hugo.Guess

import java.util.*


fun main(args: Array<String>) {
    val secrect = Random().nextInt(10)+1;
    print("Pla input number : ");
    var answer = Scanner(System.`in`).nextInt();
    while(answer != secrect){
        if(answer> secrect){
            println("Lower");
        }
        else if (answer < secrect){
            println("Higher");
        }
        else{
            println("Congratuation the answer is : $answer" )
        }
    }
}